client/components/jetpack/intro-pricing-banner/index.tsx

```diff
@@ -26,12 +27,23 @@ const CALYPSO_MASTERBAR_HEIGHT = 47;
 const CLOUD_MASTERBAR_HEIGHT = 47;
 const CONNECT_STORE_HEIGHT = 0;
 
+const useShowNoticeVAT = () => {
+	const query = useGeoLocationQuery();
+	// It's better to show more information rather than hide it. So we don't show notice if we don't have geodata yet.
```

Comment:
This comment is conflicting (I think?) To it me the first sentence implies that we *should* show the notice until the geodata is available, but the next sentence says that we shouldnt